base/math.jl

```diff
@@ -1330,8 +1353,8 @@ end
 
 function add22condh(xh::Float64, xl::Float64, yh::Float64, yl::Float64)
     # This algorithm, due to Dekker, computes the sum of two
-    # double-double numbers and returns the high double. References:
-    # [1] https://www.digizeitschriften.de/en/dms/img/?PID=GDZPPN001170007
+    # double-double numbers and return the high double. References:
+    # [1] http://www.digizeitschriften.de/en/dms/img/?PID=GDZPPN001170007
```

Comment:
Was the 's' accidentally removed? The website works with https for me.

Reply:
It did not work for me at the time of writing that commit; since that was more than half a year ago, things might have been fixed since then (an unfortunate sideeffect of PRs lingering). Note that the DOI link below still has its `https` protocol.